e2e/fixtures/auto-nav-sidebar-no-meta/doc/api/rspress-commands.mdx

```diff
@@ -0,0 +1,5 @@
+# 命令
```

Comment:
can we use English by default in e2e cases

Reply:
done